File: MobileApp/app/src/main/java/com/example/gymtracker/DTO/UserDetails.java
package com.example.gymtracker.DTO;

public class UserDetails {

    public String name;
    public String surname;
    public String email;
    public String telephoneNumber;


    public UserDetails(String name, String surname, String email, String telephoneNumber) {
        this.name = name;
        this.surname = surname;
        this.email = email;
        this.telephoneNumber = telephoneNumber;
    }
}

File: BackEnd/ClientApi/Application/Dtos/UserDetails.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ClientApi.Application.Dtos
{
    public class UserDetails
    {
        public string Name { get; set; }
        public string Surname { get; set; }
        public string Email { get; set; }
        public string TelephoneNumber { get; set; }

        public UserDetails(string name, string surname, string email, string telephoneNumber)
        {
            Name = name;
            Surname = surname;
            Email = email;
            TelephoneNumber = telephoneNumber;
        }
    }
}
